openupgrade_scripts/scripts/mail/18.0.1.18/pre-migration.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2025 Hunki Enterprises BV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from openupgradelib import openupgrade
+
+model_renames = [
+    ("mail.notification.web.push", "web.push"),
+    ("mail.partner.device", "mail.push.device"),
+    ("mail.shortcode", "mail.canned.response"),
```

Comment:
please, add this in apriori, this way next analysis will take this into account.

Reply:
see https://github.com/OCA/OpenUpgrade/pull/4898